javaScript/rot13.js

```diff
@@ -0,0 +1,14 @@
+function rot13(str) {
+    const alphabet = 'abcdefghijklmnopqrstuvwxyzabcdefghijklmABCDEFGHIJKLMNOPQRSTUVWXYZABCDEFGHIJKLM';
+    let arr = str.split('');
+    
+    for (let i = 0; i < arr.length; i++) {
```

Comment:
It is not very common to use a 'for' loop in JavaScript unless it's basically the only way.  Change this to use the `map` method. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map